tests/ui/test_views.py

```diff
@@ -134,52 +134,74 @@ def test_logged_user(app, client, superuser, admin, moderator, submitter,
     assert not res.json['metadata']['permissions']['deposits']['list']
 
 
-def test_schemas(client, admin, user, submitter, moderator):
-    """Test JSON schemas endpoint."""
-    res = client.get(url_for('sonar.schemas', record_type='documents'))
-    assert res.status_code == 200
-    assert res.json['schema']['properties'].get('organisation')
+def test_schema_no_existing(client, user):
```

Comment:
Can be replaced by `def test_non_existing_schema`, you can also change the doc string.